packages/server/graphql/mutations/helpers/validateScaleValue.ts

```diff
@@ -0,0 +1,30 @@
+import TemplateScaleValue from '../../../database/types/TemplateScaleValue'
+
+const validateColorValue = (color: string) => {
+  return /^#[0-9A-F]{6}$/i.test(color)
+}
+
+const validateScaleLabel = (label: string) => {
+  return 0 < label.length && label.length <= 2
+}
+
+const validateScaleValue = (value: number, isSpecial: boolean) => {
+  return isSpecial ? -1 <= value && value <= Math.pow(2, 31) - 1 : 0 <= value && value <= 99
```

Comment:
for special values we probably shouldn't accept a range, we should only accept those 2 values.

otherwise, some a-hole could add a scaleValue like `{value: 10000000, isSpecial: true}` and then the card would look real bad